packages/public/common-ui-web/src/components/FullscreenModal/FullscreenModal.tsx

```diff
@@ -0,0 +1,32 @@
+import { JSX } from 'react';
+import { Button } from '../Button';
+import { styles } from './FullscreenModal.styles';
+
+/**
+ * Props that can be passed to the Fullscreen Modal component.
+ */
+export interface FullscreenModalProps {
+  show: boolean;
+  onClose?: () => void;
+  title?: string;
+  children?: string | JSX.Element | JSX.Element[];
```

Comment:
remove the `children` props from the `FullscreenModalProps` interface and instead do something like this :

```typescript
export function FullscreenModal({
  show,
  onClose,
  title = '',
  children,
}: PropsWithChildren<FullscreenModalProps>) {
  ...
}
```